src/py-opentimelineio/opentimelineio/schema/timeline.py

```diff
@@ -19,33 +19,31 @@ def __repr__(self):
 
 @add_method(_otio.Timeline)
 def each_child(self, search_range=None, descended_from_type=_otio.Composable):
-    """ Generator that returns each child contained in the timeline
+    """ enerator that returns each child contained in the timeline
```

Comment:
I don't think you meant to delete the `G` here, did you?

Reply:
Nice catch. I wanted to delete the leading space, not the "G".